tests/monotouch-test/Network/NWProtocolOptionsTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+#if XAMCORE_2_0
+using Foundation;
+using Network;
+using ObjCRuntime;
+#else
+using MonoTouch.Foundation;
+using MonoTouch.Network;
+#endif
+
+using NUnit.Framework;
+
+namespace MonoTouchFixtures.Network {
+	public class NWProtocolOptionsTest {
+
+		[Test]
+		public void CreateTlsTest ()
+		{
+			TestRuntime.AssertSystemVersion (PlatformName.iOS, 12, 0, throwIfOtherPlatform: false);
+			TestRuntime.AssertSystemVersion (PlatformName.MacOSX, 10, 14, throwIfOtherPlatform: false);
```

Comment:
use the `AssertXcode` since the above is more verbose and won't pickup (correctly) tvOS